src/lib/util/Config.ts

```diff
@@ -150,14 +148,23 @@ export class Config {
 		return ecc[propertyName];
 	}
 
-	private async lookupAndUpdateToDefault(engine: ENGINE, ecc: EngineConfigContent, propertyName: string): Promise<EngineConfigContent> {
-		const defaultConfig = DEFAULT_CONFIG.engines.find(e => e.name.toLowerCase() === engine.valueOf().toLowerCase());
+	/**
+	 * Get the default ConfigContent. This value will be used as-is when a Config.json file doesn't exist, or
+	 * its contents will be merged with an existing Config.json file in cases where new values are added after the
+	 * Config.json file has been written.
+	 */
+	protected getDefaultConfig(): ConfigContent {
+		return DEFAULT_CONFIG;
+	}
+
+	protected async lookupAndUpdateToDefault(engine: ENGINE, ecc: EngineConfigContent, propertyName: string): Promise<EngineConfigContent> {
+		const defaultConfig = this.getDefaultConfig().engines.find(e => e.name.toLowerCase() === engine.valueOf().toLowerCase());
 
 		if (!ecc) { // if engine block doesn't exist, add a new one based on the default
 			ecc = defaultConfig;
 			this.addEngineToConfig(ecc);
 			this.logger.warn(`Persisting missing block for engine ${engine}`);
-		} else if (defaultConfig[propertyName]) { // engine block exists if we are here. Check if default has a value for property
+		} else if (defaultConfig[propertyName] !== undefined) { // engine block exists if we are here. Check if default has a value for property
```

Comment:
This was previously falling through to the exception when `defaultConfig[propertyName]` was `false`

Reply:
Good catch 👍